xapian-applications/omega/templates/query

```diff
@@ -106,7 +106,7 @@ $if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesiz
 </div>
 </div>
 </td>
-<td><b><a href="$html{$field{url}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
+<td><b><a href="$html{omega.cgi?FMT=$url{clicklog}&QUERYID=$url{$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}&DOCID=$url{$id}&URL=$url{$field{url}}}">$html{$or{$field{caption},$field{title},$prettyurl{$field{url}},Untitled}}</a></b><br>
```

Comment:
I believe I asked for this change not to be made to the `query` template, but rather for there to be a new template. You could alternatively do this with a conditional (similar to how we select between `inc/anyalldropbox` and `inc/anyallradio`), so that the default behaviour is not to redirect through `omega.cgi` (which requires installing an extra omegascript template, and an extra server hit — all a bit of a waste to people who don't want the clickstream data at all). Similarly, `search.log` shouldn't be created without changing the default configuration.

Also, `&` in HTML should be escaped.